readme.md

```diff
@@ -36,14 +36,34 @@ Introduction
 
 ### What's New in 4.3
 
+**OS Version Support**
+- iOS 10.0 is fully supported in Mobile SDK 5.0.
+- The minimum supported version of Xcode is now Xcode 8.
+- The minimum supported version of iOS platform is now iOS 9.0.
+
+**Other iOS Platform Enhancements**
+- Hybrid apps built on Mobile SDK will now use WKWebView instead of UIWebView.
+- Extensions are now fully supported.
+- Salesforce servers are now fully ATS compliant. As a result, we have removed ATS exceptions from Mobile SDK apps.
+
 **SmartStore Enhancements**
-- To enhance performance in certain edge cases, SmartStore adds an option for serializing unusually large soup elements in external storage.
+- We have added new APIs that allow hybrid developers to create their own named databases.
+
+**SmartSync Enhancements**
+- We now allow sync down/refresh of data stored in local soups by specifying the fields to sync.
+
+**Hybrid Networking Enhancements**
+- We have replaced forcetk.mobilesdk.js with force.js. Networking in hybrid apps is now handled natively. As a result, session token refresh happens automatically.
```

Comment:
Also we should mention smartsync.js no longer depends on jQuery but instead uses browser promises.